NetKVM/Common/ParaNdis_VirtQueue.cpp

```diff
@@ -324,7 +324,7 @@ UINT CTXVirtQueue::ReleaseTransmitBuffers(CRawCNBList &listDone)
     }
     if (i)
     {
-        NdisGetCurrentSystemTime(&m_Context->LastTxCompletionTimeStamp);
+        UpdateTimestamp(m_LastTxCompletionTimestamp);
```

Comment:
Also, can be submitted as a separate PR (or parts of refactoring/ ready for merge commits).